ntbs-service/Pages/ServiceDirectory/Region.cshtml

```diff
@@ -38,6 +38,8 @@
             var tbService = tbServiceWithCaseManagers.Key;
             var caseManagers = tbServiceWithCaseManagers.Value;
             var caseManagerTableModel = new _UserTablePartialModel {Users = caseManagers, TableId = $"case-managers-{tbService.Code}"};
+            var hospitals = Model.TbServicesWithHospitalNames.TryGetValue(tbService, out var value) ? value : null;
+            var hospitalsString = hospitals is not null ? string.Join(", ", hospitals) : null;
```

Comment:
This would be better written as a single statement:
```
var hospitalsString = Model.TbServicesWithHospitalNames.TryGetValue(tbService, out var value)
    ? string.Join(", ", hospitals)
    : null;
```